src/resources/js/components/AccountCreation.vue

```diff
@@ -103,9 +103,17 @@ export default {
         ).then((response) => {
           if (response.data.token_type === 'Bearer') {
             const token = response.data.access_token;
-            localStorage.setItem('Authorization-token', token);
+            localStorage.setItem('Authorization-token', token); // store the token in localstorage
+            // this window.axios code may be deleted when login forces a hard refresh
+            window.axios.defaults.headers.common = {
+              Authorization: `Bearer ${localStorage.getItem('Authorization-token')}`, // put it directly in header
+            };
+            this.$router.go(0); // this is a disgusting hack, it forces a refresh, and as
+            // our "current page" is the root, well it goes back to it
+            // (the correct way would have been to update the value in usertooltip
+            // through one of many way vuejs let us do that)
```

Comment:
why do you do a "disgusting hack" if you know, and explain, how to do it well ?

Reply:
because it's faster, guarantees that there are no problem (as the page is refreshed) and doesn't need to change anything in other components

Reply:
I've deleted the comments